tests/content/entity/procedural/test.js

```diff
@@ -11,6 +8,8 @@ autoTester.perform("Procedural create", Script.resolvePath("."), "secondary", fu
 
     MyAvatar.orientation = { x: 0, y: 1, z: 0, w: 0 };
 
+    var assetsRepo = "https://raw.githubusercontent.com/highfidelity/hifi_tests/" + autoTester.getBranch() + "/";
```

Comment:
this should absolute path to asset should be created from a query on the autoTester based on the repositoryPath
I would recommend to add to autoTester:
getAssetsPath() { return getRepositoryPath() + "/assets/")